Pedigrees from the 1467 MS

Eystein-Thanisch opened the pull request:
Pedigrees of various West Highland clans and noble families from the 1467 MS (NLS Adv. MS 72.1.1, fol. 1), as well as the kings of Scots and the Lords of the Isles. All sources can be found [here](https://www.1467manuscript.co.uk/index.html).

cyocum reviewed (changes requested):
Really just a bunch of questions more than specific emendations.  I am wondering about the large portion of commentary.  I wonder if we should use TrIG * in this instance which may make the comments searchable via SPARQL.  Let me know what you think.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_1.trig:
```diff
@@ -0,0 +1,414 @@
+    irishRel:genName "Ailpin";
+    irishRel:nomName "Alpin";
+    rel:childOf <#Eachdain>;
+    owl:sameAs <http://example.com/LL/rig_alban.trig#Alpín>.
```
That would certainly be legitimate. However, we could potentially add a lot of Wikidata links across IrishGen. I started adding them here for individuals from the later Middle Ages whose identity and dates are quite well-established but probably crossed the line in several places. So I'll add this link here but we should perhaps think about our policy re: Wikidata in general.

cyocum replied:
Yeah, that is fair.

Eystein-Thanisch replied:
I just realised that all the historical (?) kings of Alba already have Wikidata links in LL/rig_alban.trig. So this doesn't need to be added here after all.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_13.trig:
```diff
@@ -0,0 +1,181 @@
+    irishRel:nomName "Asmann";
+    rel:childOf [
+      a foaf:Person;
+      foaf:title "ardiarla"@sga
```
I'm not sure. We should investigate whether "ArdIarla" really is a distinctive title or whether the _ard-_ is just an honorific. Would that even affect its addition to `IrishTitles`?

Eystein-Thanisch replied:
If it turns out the _ard-_ doesn't really mean anything, would they just be `a IrishTitles:Iarla`?

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_16.trig:
```diff
@@ -0,0 +1,158 @@
+    irishRel:nomName "Gallbart";
+    rdfs:comment ".i. in fer leginn risinabartha Gallbairt";
+    rel:childOf <#GillaChatan>.
+    #The comment is written after "Gilla Chatan" but seems to relate to this person. - EPT
```
Remind me, what are the advantages and implications of having a title in the Irish Title ontology? I know we use it for kings in a line of succession, which doesn't necessarily resemble the context for all titles.

cyocum replied:
It is just a convenience for being able to search for people by their title in a simple SPARQL query.  You could do the same by searching `foaf:title` but then you would need to account for spelling variations where if you mark it via a class, you have one single spelling to contend with.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_18.trig:
```diff
@@ -0,0 +1,196 @@
+    irishRel:genName "Gilla Crist";
+    irishRel:nomName "Gilla Crist";
+    rdfs:comment "in tris ab ar sainganndrias";
+    foaf:title "ab"@sga, "abbot"@eng;
```
Quite possibly. See query above.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_20.trig:
```diff
@@ -0,0 +1,103 @@
+    irishRel:nomName "Cairell".
+    #Approximately 6 more names appear in the pedigree, for which Black provides some possible readings.
+    #However, these are so hypothetical, and the manuscript itself so difficult to make out, that I did
+    #not feel able to include anything beyond this point. - EPT
```
Ok. I'll have a look again at the MS and what Black says about it, to see how certain the estimate of six names is.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_27.trig:
```diff
@@ -0,0 +1,205 @@
+    irishRel:nomName "Ealasaid";
+    rel:childOf [
+      a foaf:Person;
+      foaf:title "mormair Comghuill"@sga, "mormaer of Cowall"@eng].
```
For titles where there is a specific polity or institution specified alongside the generic title, would we just extract the generic title for the `IrishTitles` ontology? So, in this instance, they'd be `a IrishTitles:Mormaer`, not `a IrishTitles:MormaerComghaill`.

cyocum replied:
Yes, that would be my thought.  Only generic stuff should be pulled into an ontology.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_34.trig:
```diff
@@ -0,0 +1,214 @@
+    rdfs:comment "aen mhac maith aige";
+    rdfs:seeAlso <https://www.wikidata.org/entity/Q76060800>.
+    #Black states that this individual died in 1440, but the Wikidata entry has
+    #him dying in 1400. Is this the same person? - EPT
```
Black's article gives them an obit of 1440 while the Wikidata entry for the individual who seems to be the same person (based on genealogical relationships) has 1400.

cyocum replied:
Ah right, I misread the comment and I saw 1440 twice rather than 1440 and 1400.  I would probably think that the wikidata is incorrect and Black is correct.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_28.trig:
```diff
@@ -0,0 +1,212 @@
+    a foaf:Person;
+    irishRel:genName "Alasdair Moir";
+    irishRel:nomName "Alasdair Mor".
+    #Is this the same as <#AlasdairMoir>, above? - EPT
```
The MS is pretty clear at this point. The problem is that Clann Domhnaill keeps recycling a very limited set of forenames and epithets...

cyocum replied:
Ok, that is fine :)

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_5.trig:
```diff
@@ -0,0 +1,234 @@
+    irishRel:genName "Diarmada";
+    irishRel:nomName "Diarmaid";
+    rdfs:comment "re nabartha an fer leiginn";
+    foaf:title "fer léiginn"@sga, "scholar"@eng;
```
NB I've replied to comments but not implemented changes yet.

Eystein-Thanisch commented:
NB I've replied to comments but not implemented changes yet.

Eystein-Thanisch commented on NLS.Adv.72.1.1/Highland_Pedigrees/kindred_24.trig:
```diff
@@ -0,0 +1,193 @@
+    irishRel:genName "Con Duiligh";
+    irishRel:nomName "Cu Duiligh";
+    rdfs:comment "ab Lesa Mor sanab I";
+    foaf:title "ab"@sga, "abbot"@eng;
```
> 
> 
> Really just a bunch of questions more than specific emendations. I am wondering about the large portion of commentary. I wonder if we should use TrIG * in this instance which may make the comments searchable via SPARQL. Let me know what you think.

Only just saw this. This MS certainly does require quite a lot of supporting commentary relating to the physical text, and there will doubtless be instances elsewhere in the corpus where this is also needed, so yes, maybe. Should the commentary be more formalised to be searchable?

Eystein-Thanisch commented:
> 
> 
> Really just a bunch of questions more than specific emendations. I am wondering about the large portion of commentary. I wonder if we should use TrIG * in this instance which may make the comments searchable via SPARQL. Let me know what you think.

Only just saw this. This MS certainly does require quite a lot of supporting commentary relating to the physical text, and there will doubtless be instances elsewhere in the corpus where this is also needed, so yes, maybe. Should the commentary be more formalised to be searchable, do you think?

cyocum commented:
> > Really just a bunch of questions more than specific emendations. I am wondering about the large portion of commentary. I wonder if we should use TrIG * in this instance which may make the comments searchable via SPARQL. Let me know what you think.
> 
> Only just saw this. This MS certainly does require quite a lot of supporting commentary relating to the physical text, and there will doubtless be instances elsewhere in the corpus where this is also needed, so yes, maybe. Should the commentary be more formalised to be searchable, do you think?

I wouldn't want to make it overly formal but I think it should be searchable via SPARQL or in some other way so that it doesn't get lost on ingestion to a repository.  I will do some experimentation on RDF* to see if that will work.

Eystein-Thanisch commented:
I just implemented all the above feedback and pushed to make sure checks still passed. I'll now format the curator comments as triples, after which checks will presumably start to fail until the necessary changes are pushed and this branch is rebased.

Eystein-Thanisch commented:
I've fixed the merge conflict and rebased from master but the checks are now failing for this branch. When I run Apache Jena, it returns all the comments that have now been added to the database as errors (both on this branch and elsewhere). Should we continue anyway?
